EightfoldAI/octuple
# a11y: Accessibility fix for floating dropdown

## SUMMARY:
Added Tab event for option selection, Focus border and props for Floating ui to maintain focus.

## GITHUB ISSUE (Open Source Contributors):

## JIRA TASK (Eightfold Employees Only):
Jira task - https://eightfoldai.atlassian.net/browse/ENG-113537

## CHANGE TYPE:


## TEST COVERAGE:


## TEST PLAN:

- With NVDA on, go to Select component and add dropDown props with "portal = true".
- use arrow keys to navigate between option and observe if there is a focus border visible.
- Use Enter, Space and Tab to select the option.
